rollup.utils.js

```diff
@@ -7,23 +7,6 @@ const path = require("node:path");
 
 const REPO_ROOT_DIR = __dirname;
 
-let activeOutputDir = "build";
-if (process.env.LOCAL_BUILD_DIRECTORY) {
-  let appDir = path.resolve(process.env.LOCAL_BUILD_DIRECTORY);
-  try {
-    fse.readdirSync(path.join(appDir, "node_modules"));
-  } catch {
-    console.error(
-      "Oops! You pointed LOCAL_BUILD_DIRECTORY to a directory that " +
-        "does not have a node_modules/ folder. Please `npm install` in that " +
-        "directory and try again."
-    );
-    process.exit(1);
-  }
-  console.log("Writing rollup output to", appDir);
-  activeOutputDir = appDir;
-}
```

Comment:
I think we lose the `yarn watch` functionality if we do this since it calls rollup directly.  There's a `watch.onEnd` script we could add the copy to though if we go this route